www/about/policies/terms-of-service.md

```diff
@@ -0,0 +1,414 @@
+receive payments via the Service, you agree to their terms of service.
+
+1. Nonprofits and Charities using Gratipay
+
+  1. If your Team is associated with a nonprofit organization, soliciting
+     payments on Gratipay may subject you to certain state laws regarding
+charitable solicitations, including various registration, reporting, and audit
+requirements. You are solely responsible for compliance with these and any
+other applicable law or regulation.
+
+  1. Gratipay makes no representation or warranty regarding the tax-exempt
+     status of any Team, or the tax-deductible status of any payment made via
+the Service.
+
+1. Copyright Policy
+
+  1. Gratipay responds to reports of copyright infringement according to the
+     notice-and-takedown procedure provided by the Digital Millennium Copyright
+Act ("DMCA"). You can contact our designated agent for copyright infringement
+notifications and counter-notifications at: Copyright Agent, Gratipay, LLC, 716
+Park Road, Ambridge, PA 15003, USA, legal@gratipay.com.
+
+  1. Reporting Instances of Copyright Infringement 
+
+    1. If you believe that any content on the Service infringes your copyright,
+       please send us a notice containing the following information: a)
+identification of the work or material being infringed; b) identification of
+the material that is claimed to be infringing, including its location, with
+sufficient detail so that Gratipay is capable of finding it and verifying its
+existence; c) your contact information, including name, address, telephone
+number, and email address; d) a statement that you have a good faith belief
+that the material is not authorized by the copyright owner, its agent, or the
+law; e) a statement made under penalty of perjury that the information provided
+in the notice is accurate and that you are authorized to make the complaint on
+behalf of the copyright owner; f) a physical or electronic signature of a
+person authorized to act on behalf of the owner of the copyright that has been
+allegedly infringed.
+
+    1. Infringement notices are not confidential. Gratipay may provide the
+       information contained in a notice to the Participant who posted the
+allegedly infringing content, or to other parties.
+
+  1. Submitting a DMCA Counter-Notification
+
+    1. If you believe that content you posted was wrongfully removed or
+       disabled as a result of a copyright infringement notice, you may notify
+Gratipay by providing the following information: a) the specific location of
+the material that Gratipay has removed or disabled access to; b) your name,
+address, telephone number, and email address; c) a statement that you consent
+to the jurisdiction of Federal District Court for the judicial district in
+which your address is located (or the federal district courts located in New
+York County, New York if your address is outside of the United States), and
```

Comment:
Since we changed the venue from New York to Pennsylvania in https://github.com/gratipay/gratipay.com/commit/3e15561cfbfb6941195755c9222159d6d93e8883, do we also want to change it here?